app/src/main/java/com/relaypro/app/examples/multi/DeviceInfoWorkflow.java

```diff
@@ -27,8 +27,7 @@ public void onInteractionLifecycle(Relay relay, InteractionLifecycleEvent lifecy
 
         if (lifecycleEvent.isTypeStarted()) {
             relay.setDeviceName(interactionUri, "optimus prime");
-            relay.setLocationEnabled(interactionUri, true);
```

Comment:
Maybe turn this into sayAndWait("location services are " + relay.getLocationEnabled() ? "enabled" : "disabled")

Reply:
Good idea - fixed in latest commit.